utils/nativefuncs.go

```diff
@@ -242,6 +243,33 @@ func RegisterNativeFuncs(vm *jsonnet.VM, resolver Resolver) {
 			return ret, nil
 		},
 	})
+
+	vm.NativeFunction(&jsonnet.NativeFunction{
+		Name:   "validateJSONSchema",
+		Params: []jsonnetAst.Identifier{"obj", "schema"},
+		Func: func(args []interface{}) (interface{}, error) {
+			obj := args[0].(map[string]interface{})
+			schema := args[1].(map[string]interface{})
```

Comment:
what's the purpose of these "casts" ?

both `json.Marshal` and `.Validate` below accept `interface{}`

```suggestion
			obj := args[0]
			schema := args[1]
```

Reply:
Agreed, I was going for readability, but there's no need to have the trailing `map[string]interface{}` casting.